Release 2024.12.0 (#5238)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/release-notes/2024/index.md

| [System Log event for emails added to the bounced email list](#system-log-event-for-emails-added-to-the-bounced-email-list) | December 11, 2024 |
| [Granular account linking for certain Identity Providers is self-service EA](#granular-account-linking-for-certain-identity-providers-is-self-service-ea) | December 11, 2024 |
| [Request access option removed from the OIN Manager](#request-access-option-removed-from-the-oin-manager) | December 11, 2024 |
| [SCIM documentation link updated in the OIN Wizard](#scim-documentation-link-updated-in-the-oin-wizard) | December 11, 2024 |
| [Use case selection in the OIN Wizard](#use-case-selection-in-the-oin-wizard) | December 11, 2024 |
| [Industry term update in the OIN catalog](#industry-term-update-in-the-oin-catalog) | December 11, 2024 |
| [Developer documentation updates in 2024.12.0](#developer-documentation-updates-in-2024-12-0) | December 11, 2024 |
| [Bug fixed in 2024.12.0](#bug-fixed-in-2024-12-0) | December 11, 2024 |

#### Restrict access to the Admin Console is EA in Preview

> By default, users and groups with assigned admin roles have access to the Admin Console app. With this feature, super admins can choose to manually assign the app to delegated admins instead. This is recommended for orgs with admins who don't need access, like business partners, third-party admins, or admins who only use the Okta API. See [Configure administrator settings](https://help.okta.com/okta_help.htm?type=oie&id=administrator-settings) and the corresponding APIs: [Retrieve the Okta Admin Console Assignment Setting](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/OrgSettingAdmin/#tag/OrgSettingAdmin/operation/getAutoAssignAdminAppSetting) and [Update the Okta Admin Console Assignment Setting](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/OrgSettingAdmin/#tag/OrgSettingAdmin/operation/updateAutoAssignAdminAppSetting). <!-- OKTA-717742 ADMIN_APP_AND_ROLE_DECOUPLING -->

Reviewer: ```suggestion
By default, users and groups with assigned admin roles have access to the Admin Console app. With this feature, super admins can choose to manually assign the app to delegated admins instead. This is recommended for orgs with admins who don't need access, like business partners, third-party admins, or admins who only use the Okta API. See [Configure administrator settings](https://help.okta.com/okta_help.htm?type=oie&id=administrator-settings) and the corresponding APIs: [Retrieve the Okta Admin Console assignment setting](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/OrgSettingAdmin/#tag/OrgSettingAdmin/operation/getAutoAssignAdminAppSetting) and [Update the Okta Admin Console assignment setting](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/OrgSettingAdmin/#tag/OrgSettingAdmin/operation/updateAutoAssignAdminAppSetting). <!-- OKTA-717742 ADMIN_APP_AND_ROLE_DECOUPLING -->
```

#### New skipping of entitlement sync during import of a user Systems Log event

A new System Log event has been added to indicate when an entitlement was skipped while importing a user. <!--OKTA-828548-->

#### Create dynamic resource sets with conditions is EA in Production

Resource set conditions help you limit the scope of a role by excluding an admin's access to certain apps. This gives you more granular control over your custom admin roles and helps meet your org's unique security needs. See [Resource set conditions](https://help.okta.com/okta_help.htm?type=oie&id=resource-set-conditions) and the corresponding [Resource Set Resources](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/RoleCResourceSetResource/) API. <!-- OKTA-746719 DYNAMIC_RESOURCE_SETS -->

#### System Log event for emails added to the bounced email list

> A System Log `system.email.bounce.removal` event is now triggered when an API request is made to [remove bounced emails](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/EmailCustomization/#tag/EmailCustomization/operation/bulkRemoveEmailAddressBounces) (`POST /org/email/bounces/remove-list`). This request sends a list of emails to a third-party email service to remove the emails from the bounce list. The event is triggered when the API request is made. The event doesn’t indicate when the emails are actually removed by the third-party email service. <!--OKTA-807544 OKTA-754648 -->

Reviewer: ```suggestion
A System Log `system.email.bounce.removal` event is now triggered when an API request is made to [remove bounced emails](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/EmailCustomization/#tag/EmailCustomization/operation/bulkRemoveEmailAddressBounces) (`POST /org/email/bounces/remove-list`). This request sends a list of emails to a third-party email service to remove the emails from the bounce list. The event is triggered when the API request is made. The event doesn't indicate when the emails are actually removed by the third-party email service. <!--OKTA-807544 OKTA-754648 -->
```

#### Granular account linking for certain Identity Providers is self-service EA

> When admins link users from SAML and OIDC Identity Providers, they can now exclude specific users and admins. This improves security by allowing admins to configure granular access control scenarios. See Add an External Identity Provider [OpenId Connect](/docs/guides/add-an-external-idp/openidconnect/main/#create-an-identity-provider-in-okta) and [SAML 2.0](/docs/guides/add-an-external-idp/saml2/main/#create-an-identity-provider-in-okta) <!-- EXTENDED_ACCOUNT_LINKING_FILTERS OKTA-831244-->

Reviewer: ```suggestion
When admins link users from SAML and OIDC Identity Providers, they can now exclude specific users and admins. This improves security by allowing admins to configure granular access control scenarios. See [Add an external Identity Provider for OpenId Connect](/docs/guides/add-an-external-idp/openidconnect/main/#create-an-identity-provider-in-okta) and [SAML 2.0](/docs/guides/add-an-external-idp/saml2/main/#create-an-identity-provider-in-okta) <!-- EXTENDED_ACCOUNT_LINKING_FILTERS OKTA-831244-->
```

Author: I think this one is formatted this way because there's the link for OpenID Connect and for SAML 2.0 separately. I'm going to leave this one as is.

> When admins link users from SAML and OIDC Identity Providers, they can now exclude specific users and admins. This improves security by allowing admins to configure granular access control scenarios. See **Add an external Identity Provider** for [OpenId Connect](/docs/guides/add-an-external-idp/openidconnect/main/#create-an-identity-provider-in-okta) and [SAML 2.0](/docs/guides/add-an-external-idp/saml2/main/#create-an-identity-provider-in-okta) <!-- EXTENDED_ACCOUNT_LINKING_FILTERS OKTA-831244-->

Author: ```suggestion
When admins link users from SAML and OIDC Identity Providers, they can now exclude specific users and admins. This improves security by allowing admins to configure granular access control scenarios. See **Add an external Identity Provider** for [OpenId Connect](/docs/guides/add-an-external-idp/openidconnect/main/#create-an-identity-provider-in-okta) and [SAML 2.0](/docs/guides/add-an-external-idp/saml2/main/#create-an-identity-provider-in-okta). <!-- EXTENDED_ACCOUNT_LINKING_FILTERS OKTA-831244-->
```

#### Request access option removed from the OIN Manager

> You can no longer request access to your org’s OIN Manager account with the **Request Access** button from the OIN Manager access-denied error page. If you need access to your org’s OIN Manager, contact your Okta admin to grant you the appropriate permissions. <!--OKTA-840117-->

Reviewer: ```suggestion
You can no longer request access to your org's OIN Manager account with the **Request Access** button from the OIN Manager access-denied error page. If you need access to your org's OIN Manager, contact your Okta admin to grant you the appropriate permissions. <!--OKTA-840117-->
```
